zwave_js_server/model/node.py

```diff
@@ -325,7 +325,7 @@ def receive_event(self, event: Event) -> None:
 
         self.emit(event.type, event.data)
 
-    async def _async_send_command(
+    async def async_send_command(
```

Comment:
Why are we changing these?

Reply:
I changed this because I access this method in the new utility function. The method can't be private as a result